Translation corrections (#480)
07th-mod/higurashi-console-arcs: Update/omot_022.txt

	//r…時間的に言っても、無茶どころか非常識な頼みだった。せいぜい夕方くらいに届けてくれたら拝み倒して感謝しようと、思っていたくらいなのだ。
		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
		OutputLine(NULL, "…時間的に言っても、無茶どころか非常識な頼みだった。せいぜい夕方くらいに届けてくれたら拝み倒して感謝しようと、思っていたくらいなのだ。",

> 				NULL, "Given the timing, it wasn't just an unreasonable request. It was absolutely insane. So much that he would've been deeply appreciative even if she hadn't delivered this notice until much later this evening.", Line_Normal);

Reviewer: ```suggestion
				NULL, "Given the timing, it wasn't just an unreasonable request. It was absolutely insane. He would've been deeply appreciative even if she hadn't delivered this notice until much later this evening.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#c89a80>入江</color>", NULL, "<color=#c89a80>Irie</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 10, "ps3/s14/10/22100270", 256, TRUE);
	OutputLine(NULL, "「こ、こんな狼藉(ろうぜき)が許されるとでも…ぐっ？！」",

> 			NULL, "\"D-do you think this sort of violence will bet per... Ngh?!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"D-do you think this sort of violence will bet permi... Ngh?!\"", Line_Normal);
```

//r沙都子も眠ってしまい、退屈な時間が続いてあくびをしかけたそのときだった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "沙都子も眠ってしまい、退屈な時間が続いてあくびをしかけたそのときだった。",

> 			NULL, "It happened when Satoko had fallen asleep, and I was starting to yawn at spending such a boring time here.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It happened after Satoko had fallen asleep, and just as I began to yawn at spending such a boring time here.", Line_Normal);
```

	if (GetLocalFlag(LConsoleArc)==10) {
		ModPlayVoiceLS(4, 4, "ps2/04/220400141", 540, TRUE);
		OutputLine(NULL, "「そんなにこの小娘が恐ろしいのでございますかッ？！",

> 				NULL, "\"Are you that scared of \"this girl\"?!", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, "\"Are you that afraid of this little girl?!", Line_WaitForInput);
```
